lib/migrate/MigrationGenerator.js

```diff
@@ -30,9 +30,7 @@ class MigrationGenerator {
     const dirs = this._absoluteConfigDirs();
 
     const promises = dirs.map((dir) => {
-      return bluebird
-        .promisify(fs.stat, { context: fs })(dir)
-        .catch(() => bluebird.promisify(mkdirp)(dir));
+      return promisify(fs.stat)(dir).catch(() => promisify(mkdirp)(dir));
```

Comment:
`{ context: fs }` part is irrelevant?

Reply:
yes, but i cant find it in official docs
https://nodejs.org/docs/latest-v8.x/api/util.html#util_util_promisify_original